### ISSUE TITLE
Dedicated Kiwix hotspot stickers for RPIs

### ISSUE THREAD

**Issue by kelson42** [kiwix/web#41]
Having a look to http://www.kiwix.org/kiwix-plug/ and the pictures of RPIs with stickers, it appears that we might achieve a better branding of the hardware boxes with Kiwix hotspot by just having a sticker for the case which:
* Cover the whole top of the case
* Has a nice design
* Provide a blank zone where people can write the wifi SSID/name

**Comment by Popolechien** [comment]
Ok here is what Stefan prepared:
<img width="899" alt="screen shot 2018-10-12 at 16 16 24" src="https://user-images.githubusercontent.com/21675885/46874836-699b2380-ce3a-11e8-9394-18fff5a7b910.png">
But then I realized that those RPi3 would only be marketed/sold by us, in which case the SSID is more than likely to be "kiwix" and there won't be a password either. Having a "password" field for something that is at best a fringe case might end up being a bit confusing for the end user, wouldn't it?

So should we really go down this route?

**Comment by rgaudin** [comment]
I'd also prefer something that covers the whole top of the Pi ; would look better IMO.

I am also in favor of more colors as Pis are usually left on dusty places so white would get dirty.

As for Password, I think the best is to add a way to mark clearly whether it's an Open Network or a password-protected one and in that case have the password.

We could send one of those with every card we sell from the cardshop…

**Comment by Popolechien** [comment]
@kelson42 The short answer to 1) is time+money; 2) time+talent. We'll try to come up with something in the next few days and come back here.

**Comment by Popolechien** [comment]
Yes, not as fancy as you'd have hoped but fixes the issue for the time being: it's essentially a QR code with a logo and the goto.kiwix.hotspot URL also written. 
We chose against getting a full-cover design because of the possibility to use it on RaspberryPi0 as well (and also: cost of a custom size vs. standardized). I've placed a small order, we'll see what gives.

**Comment by Popolechien** [comment]
[autocollant hotspot.pdf](https://github.com/kiwix/web/files/2554279/autocollant.hotspot.pdf)
